### PR TITLE
New Repo: cloud-provider-ibm

### PR DESCRIPTION
**Pull request by Fedosin**
Generated by `make new-repo`

### CODE REVIEW

**Review by rtheis — APPROVED** [review]

**Review by rtheis — COMMENTED** [review]
Is `ci-operator/jobs/openshift/cloud-provider-ibm/OWNERS` supposed to be empty?

**Comment by Fedosin** [comment]
@rtheis no... it's not critical, because it will be generated automatically later, but it's better to fill it here.

**Comment by Fedosin** [comment]
thanks for noticing!

File: ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
```
Comment (jonesbr17) [inline ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml:38]: Not really sure what the scope of this should be, but we have a full unit test and fvt that could be run here as well

**Review by jonesbr17 — COMMENTED** [review]
Closed my PR in favor of this one

File: ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml (hunk 1 above)
Comment (Fedosin) [reply ci-operator/config/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master.yaml:38]: Yep, I added unit tests and fmt. fvt requires an additional binary, so I'd prefer to skip this job. 
I think that what we have now is enough for a downstream fork. Later we will add an e2e job for IBM Cloid.

File: ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-postsubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    cluster: build02
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/is-promotion: "true"
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    max_concurrency: 1
+    name: branch-ci-openshift-cloud-provider-ibm-master-images
+    path_alias: github.com/IBM-Cloud/cloud-provider-ibm
```
Comment (elmiko) [inline ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-postsubmits.yaml:15]: shouldn't this be the repo in openshift?

File: ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    cluster: build01
+    context: ci/prow/fmt
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cloud-provider-ibm-master-fmt
+    path_alias: github.com/IBM-Cloud/cloud-provider-ibm
```
Comment (elmiko) [inline ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml:16]: openshift?

File: ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    cluster: build01
+    context: ci/prow/images
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cloud-provider-ibm-master-images
+    path_alias: github.com/IBM-Cloud/cloud-provider-ibm
```
Comment (elmiko) [inline ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml:65]: openshift?

File: ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+    cluster: build01
+    context: ci/prow/test
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cloud-provider-ibm-master-test
+    path_alias: github.com/IBM-Cloud/cloud-provider-ibm
```
Comment (elmiko) [inline ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml:115]: openshift?

File: ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+    cluster: build01
+    context: ci/prow/vet
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cloud-provider-ibm-master-vet
+    path_alias: github.com/IBM-Cloud/cloud-provider-ibm
```
Comment (elmiko) [inline ci-operator/jobs/openshift/cloud-provider-ibm/openshift-cloud-provider-ibm-master-presubmits.yaml:164]: openshift?

**Review by elmiko — COMMENTED** [review]
generally looks good, but i'm a little confused by some of the repo fields.

**Comment by Fedosin** [comment]
@elmiko as I understand for forks we use the original repo location for path aliases.

Here is what we do for cluster-provider-aws: https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/cluster-api-provider-aws/openshift-cluster-api-provider-aws-master-presubmits.yaml#L14

**Comment by elmiko** [comment]
> @elmiko as I understand for forks we use the original repo location for path aliases.

TIL !

/lgtm

**Pull request merged**